rkovalenko/3Sum.cs

```diff
@@ -0,0 +1,44 @@
+public class Solution {
+    public IList<IList<int>> ThreeSum(int[] nums) {
+        Array.Sort(nums);
+        var hashSet = new HashSet<(int,int,int)>();
```

Comment:
Good idea to use hashset of tuples. I've tried using ordinary hashsets for triplets and duplicates were a problem like [0,0,0] shortened to just [0]. Tuples are good things to have here. :)